src/schema/json/Patient.json

```
@@ -1,23 +1,68 @@
-// Extracted from CCDH MVPv0
 {
     "$id": "https://example.org/cda-data-model/Patient#",
-    "$schema": "http://json-schema.org/draft-07/schema#",
-    "Patient": {
-        "additionalProperties": false,
-        "description": "",
-        "properties": {
+    "$schema": "http://json-schema.org/draft-07/schema#/",   
+    "description": "A patient entity captures the study-independent metadata for research subjects. Human research subjects are usually not traceable to a particular person to protect the subject\u2019s privacy.",
+    "definitions": {},
+    "additionalProperties": true,
+    "properties": {
+        "describedBy": {
+            "description": "The JSON Schema used for this object.",
+            "type": "string"
+        },
         "id": {
+            "description": "The 'logical' identifier of the entity in the system of record, e.g. a UUID.  This 'id' is unique within a given system. The identified entity may have a different 'id' in a different system.",
             "type": "string"
         },
+        "identifier": {
+            "description": "A 'business' identifier for the entity, typically as provided by an external system or authority, that persists across implementing systems  (i.e. a  'logical' identifier). Uses a specialized, complex 'Identifier' data type to capture information about the source of the business identifier - or a URI expressed as a string to an existing entity. ",
+            "items": {
+                "oneOf" : [
+                     {"$ref": "#/definitions/Identifier"},
+                     {"type": "string"}
+                 ]            
+            },
+            "type": "array"
+         },
         "taxon": {
-            "$ref": "#/definitions/CodeableConcept",
-            "description": "The taxonomic group (e.g. species) of the patient."
-         }
+            "description": "The taxonomic group (e.g. species) of the patient. For MVP, since taxonomy vocabulary is consistent between GDC and PDC, using text.  Ultimately, this will be a term returned by the vocabulary service.",
+            "oneOf" : [
+                {"$ref": "#/definitions/CodeableConcept"},
+                {"type": "string"}
+            ]
+        },
+        "research_subject_list": {
+            "description": "A list of ResearchSubject identities for this Patient.  We could keep a list of the Projects but the ResearchSubject mimics the use of case_id in existing data.  All relevant data for each Project will be in the ResearchSubject.",
+            "items": {
+                "research_subject_id": {
+                    "oneOf" : [
+                        {"$ref": "https://example.org/cda-data-model/ResearchSubject.json"},
```

Is this the correct syntax to refer to ResearchSubject? I would expect something like
```suggestion
                        {"$ref": "#/ResearchSubject/ResearchSubject"},
```

---

https://broadworkbench.atlassian.net/browse/CDA-58 

And yes, I use two validators online; they're mentioned in the CDA ticket 
Using these 2 validators …

https://www.jsonschemavalidator.net/

https://extendsclass.com/json-schema-validator.html

---

Hmm, I don't think so but since the schemas aren't published, I can't really test this.